programs/mango-v4/src/instructions/openbook_v2_close_open_orders.rs

```diff
@@ -0,0 +1,71 @@
+use anchor_lang::prelude::*;
+
+use openbook_v2::cpi::accounts::CloseOpenOrdersAccount;
+
+use crate::accounts_ix::*;
+use crate::error::MangoError;
+use crate::state::*;
+
+pub fn openbook_v2_close_open_orders(ctx: Context<OpenbookV2CloseOpenOrders>) -> Result<()> {
+    //
+    // Validation
+    //
+    let mut account = ctx.accounts.account.load_full_mut()?;
+    // account constraint #1
+    require!(
+        account
+            .fixed
+            .is_owner_or_delegate(ctx.accounts.authority.key()),
+        MangoError::SomeError
+    );
+
+    let openbook_market = ctx.accounts.openbook_v2_market.load()?;
+
+    // Validate open_orders #2
+    require!(
+        account
+            .openbook_v2_orders(openbook_market.market_index)?
+            .open_orders
+            == ctx.accounts.open_orders_account.key(),
+        MangoError::SomeError
+    );
+
+    //
+    // close OO
+    //
+    let account_seeds = mango_account_seeds!(account.fixed);
+    cpi_close_open_orders(ctx.accounts, &[account_seeds])?;
+
+    // Reduce the in_use_count on the token positions - they no longer need to be forced open.
+    // We cannot immediately dust tiny positions because we don't have the banks.
```

Comment:
this points towards us possibly passing the banks here now that we have the opportunity